pyproject.toml

```diff
@@ -37,6 +37,7 @@ openpyxl = {version="^3.1.2", optional=true}
 pydataxm = {version="^0.3.2", optional=true}
 ruamel-yaml = "^0.17.24"
 odfpy = {version = "^1.4.1", optional = true}
+pycountry = "^22.3.5"
```

Comment:
This should probably be added as an optional dependency in the parser group to match the others? (or even a new capacity parser group)